ruby/private/bundle/bundle.bzl

```diff
@@ -13,68 +13,71 @@ def _get_bundler_lib_label(repository_ctx, ruby_sdk):
     return Label("%s//:bundler/lib" % ruby_sdk)
 
 def bundle_install_impl(ctx):
-  ctx.symlink(ctx.attr.gemfile, "Gemfile")
-  ctx.symlink(ctx.attr.gemfile_lock, "Gemfile.lock")
-  ctx.symlink(ctx.attr._create_bundle_build_file, "create_bundle_build_file.rb")
-  if ctx.attr.gemspec:
-    ctx.symlink(ctx.attr.gemspec, ctx.path(ctx.attr.gemspec).basename)
+    ctx.symlink(ctx.attr.gemfile, "Gemfile")
+    ctx.symlink(ctx.attr.gemfile_lock, "Gemfile.lock")
+    ctx.symlink(ctx.attr._create_bundle_build_file, "create_bundle_build_file.rb")
+    if ctx.attr.gemspec:
+        ctx.symlink(ctx.attr.gemspec, ctx.path(ctx.attr.gemspec).basename)
 
-  ruby = _get_interpreter_label(ctx, ctx.attr.ruby_sdk)
-  bundler = _get_bundler_label(ctx, ctx.attr.ruby_sdk)
+    ruby = _get_interpreter_label(ctx, ctx.attr.ruby_sdk)
+    bundler = _get_bundler_label(ctx, ctx.attr.ruby_sdk)
 
-  # Install the Gems into the workspace
-  args = [
-      'env', '-i',                                              # remove all environment variables
-      ctx.path(ruby),                                           # ruby
-      '--disable-gems',                                         # prevent the addition of gem installation directories to the default load path
-      '-I', ctx.path(bundler).dirname.dirname.get_child('lib'), # Used to tell Ruby where to load the library scripts
-      ctx.path(bundler),                                        # execute bundler
-      'install',                                                # install
-      '--deployment',                                           # In deployment mode, Bundler will 'roll-out' the bundle for production or CI use.
-      '--standalone',                                           # Makes a bundle that can work without depending on Rubygems or Bundler at runtime.
-      '--frozen',                                               # Do not allow the Gemfile.lock to be updated after this install.
-      '--binstubs=bin',                                         # Creates a directory and place any executables from the gem there.
-      '--path=lib',                                             # The location to install the specified gems to.
-  ]
-  result = ctx.execute(args, quiet=False)
+    # Install the Gems into the workspace
+    args = [
+        "env",
```

Comment:
Same here — please keep all actual Ruby Code as is (formatted by rubocop), but realing comments lioke you had it before. and then wrap the entire block in the rubcop-chil comment block.

Reply:
I think we can do that later, this is also not rubocop this is buildifier, I am not sure how to ask that not to apply styles